src/background/services/tabEvents.ts

```diff
@@ -179,7 +179,7 @@ export class TabEvents {
     state: Storage['state'];
     tabInfo: PopupTabInfo;
   }) {
-    let title = this.t('appName');
+    let { name: title } = this.browser.runtime.getManifest();
```

Comment:
I wonder if I should the name (as `appName`) in `Cradle` to ensure we don't have to getManifest repeatedly?